### PR TITLE
Update Release Responsible Plan

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area open-source
/kind enhancement

**What this PR does / why we need it**:
Update Release Responsible Plan

**Release note**:

```other operator
NONE
```

/invite @vpnachev @voelzmo @plkokanov @kris94 @danielfoehrKn @timebertt @ialidzhikov @timuthy @BeckerMax @stoyanr @vanjiii
/assign @vpnachev @voelzmo @plkokanov @kris94 @danielfoehrKn @timebertt @ialidzhikov @timuthy @BeckerMax @stoyanr @vanjiii

### CODE REVIEW

**Review by vpnachev — DISMISSED** [review]
/lgtm
/unassign

**Review by ialidzhikov — DISMISSED** [review]
/lgtm
/unassign

**Review by timuthy — DISMISSED** [review]
/lgtm
/unassign
Thank you very much

**Review by plkokanov — DISMISSED** [review]
/lgtm
/unassign

**Review by danielfoehrKn — DISMISSED** [review]

File: docs/development/process.md [hunk 1]
```diff
@@ -12,31 +12,48 @@ The [@gardener-maintainers](https://github.com/orgs/gardener/teams/gardener-main
...
-v1.28   | Week 29-30  | July 19, 2021          | August 1, 2021     | [@ialidzhikov](https://github.com/ialidzhikov)     |
-v1.29   | Week 31-32  | August 2, 2021         | August 15, 2021    | [@timuthy](https://github.com/timuthy)             |
-v1.30   | Week 33-34  | August 16, 2021        | August 29, 2021    | [@BeckerMax](https://github.com/BeckerMax)         |
-v1.31   | Week 35-36  | August 30, 2021        | September 12, 2021 | [@stoyanr](https://github.com/stoyanr)             |
 v1.32   | Week 37-38  | September 13, 2021     | September 26, 2021 | [@vpnachev](https://github.com/vpnachev)           |
 v1.33   | Week 39-40  | September 27, 2021     | October 10, 2021   | [@voelzmo](https://github.com/voelzmo)             |
+v1.34   | Week 41-42  | October 11, 2021       | October 24, 2021   | [@kris94](https://github.com/kris94)               |
+v1.35   | Week 43-44  | October 25, 2021       | November 7, 2021   | [@plkokanov](https://github.com/plkokanov)         |
+v1.36   | Week 45-46  | November 8, 2021       | November 21, 2021  | [@rfranzke](https://github.com/rfranzke)           |
```
Comment (timebertt) [inline docs/development/process.md]:
```suggestion
v1.36   | Week 45-46  | November 8, 2021       | November 21, 2021  | [@timebertt](https://github.com/timebertt)           |
```

File: docs/development/process.md [hunk 2]
```diff
@@ -12,31 +12,48 @@ The [@gardener-maintainers](https://github.com/orgs/gardener/teams/gardener-main
...
+v1.37   | Week 47-48  | November 22, 2021      | December 5, 2021   | [@danielfoehrKn](https://github.com/danielfoehrKn) |
+v1.38   | Week 49-50  | December 6, 2021       | December 19, 2021  | [@timebertt](https://github.com/timebertt)         |
```
Comment (timebertt) [inline docs/development/process.md]:
```suggestion
v1.38   | Week 49-50  | December 6, 2021       | December 19, 2021  | [@rfranzke](https://github.com/rfranzke)         |
```

**Review by timebertt — COMMENTED** [review]
Week 49-50 might not work for me. I propose to switch with @rfranzke (Week 45-46) to be on the safe side.

**Review by timebertt — DISMISSED** [review]
Thanks!
/lgtm
/unassign

**Review by stoyanr — DISMISSED** [review]
/lgtm

**Pull request merged**